### PR TITLE
Test improvements

### PR DESCRIPTION
**Pull request by jgebal**
- Updated test to use --%throws annotation instead of capturing exception in code
- Changed expectations syntax from  `to_( equal() )` to `to_equal()`
- Removed unneeded `--%displayname` annotations

### CODE REVIEW

File: test/award_bonus/test_award_bonus.pkb [hunk 1]
```diff
@@ -10,37 +10,36 @@ create or replace package body test_award_bonus as
...
     c_sales_amount constant number := 1000;
   begin
-    --arrange
+    --Arrange
     open expected for
       select (salary + c_sales_amount * gc_commision_pct) as new_salary
         from employees_test where employee_id = gc_test_employee;
 
     open not_affected for
       select * from employees_test where employee_id <> gc_test_employee;
```
Comment (erikvanroon) [inline test/award_bonus/test_award_bonus.pkb:20]: Erik: Changing that line-comment to initcap is going to be the killer enhancement for utPLSQL :)

**Review by erikvanroon — COMMENTED** [review]
Hope this is what you need?

File: test/award_bonus/test_award_bonus.pkb (hunk 1 above)
Comment (jgebal) [reply test/award_bonus/test_award_bonus.pkb:20]: Perfect - Thank you @erikvanroon

File: test/award_bonus/test_award_bonus.pkb [hunk 2]
```diff
@@ -10,37 +10,36 @@ create or replace package body test_award_bonus as
...
 
-    --act
+    --Act
```
Comment (jgebal) [inline test/award_bonus/test_award_bonus.pkb:21]: @erikvanroon Better to comment on new code than the old code 😄

**Pull request closed without merging**